drivers/dma/hsu/pci.c

```diff
@@ -105,6 +96,16 @@ static int hsu_pci_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 	if (ret)
 		goto err_register_irq;
 
+        /*
```

Comment:
Indentation is inconsistent with all comment lines (seems like TABs/spaces mixture.

Reply:
Yes. Seems my editor is to insert spaces for tab. Am I right the correct settings should be mixed, indent 4, tab = 8?